enterprise/payara.tooling/nbproject/org-netbeans-modules-payara-tooling.sig

```diff
@@ -508,7 +508,7 @@ CLSS public org.netbeans.modules.payara.tooling.admin.CommandVersion
 cons public init()
 meth public static boolean verifyResult(org.netbeans.modules.payara.tooling.admin.ResultString,org.netbeans.modules.payara.tooling.data.PayaraServer)
 meth public static org.netbeans.modules.payara.tooling.admin.ResultString getVersion(org.netbeans.modules.payara.tooling.data.PayaraServer)
-meth public static org.netbeans.modules.payara.tooling.data.PayaraVersion getPayaraVersion(org.netbeans.modules.payara.tooling.data.PayaraServer)
+meth public static org.netbeans.modules.payara.tooling.data.PayaraPlatformVersionAPI getPayaraVersion(org.netbeans.modules.payara.tooling.data.PayaraServer)
```

Comment:
Keep original and introduce new method. Probably easiest, if you choose different name.